packages/testsuite/cypress.config.ts

```diff
@@ -29,48 +28,138 @@ export default defineConfig({
                 },
               ])
               .withWaitStrategy(Wait.forLogMessage(new RegExp(".*(WildFly Full.*|JBoss EAP.*)started in.*")))
-              .withStartupTimeout(333000)
-              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[])
+              .withStartupTimeout(30000);
```

Comment:
This is a lot lower timeout. The original value was safe even for downloading all the images. Are you sure this new value is OK?

Reply:
value reverted.